pkgs/development/python-modules/nicegui-highcharts/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  setuptools,
+  docutils,
+  nicegui,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "nicegui-highcharts";
+  version = "2.0.2";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "zauberzeug";
+    repo = "nicegui-highcharts";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-r4X4faU7Nlq/FDbIYbTpvnC1w14XskpsNGtkEXtGrFo=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail "setuptools>=30.3.0,<50" "setuptools"
```

```suggestion
      --replace-fail "setuptools>=30.3.0,<50" ""
```
I don't think they actually use setuptools.
https://github.com/search?q=repo%3Azauberzeug%2Fnicegui-highcharts%20setuptools&type=code